packages/commons-ui-next/.storybook/main.js

```diff
@@ -0,0 +1 @@
+module.exports = require("../../ui/.storybook/main");
```

Comment:
If this is shared then it should be moved to a shareable package config.

Reply:
Yeah, I thought as well. @kilemensi I will work on that

Reply:
Fixed